### PR TITLE
Add plan9port rc shell support

### PR DESCRIPTION
**Pull request by pocket7878**
Add plan9port rc shell support to `hub alias`

### CODE REVIEW

File: commands/alias.go [hunk 1]
```diff
@@ -92,6 +94,8 @@ func alias(command *Command, args *Args) {
 			profile = "~/.cshrc"
 		case "tcsh":
 			profile = "~/.tcshrc"
+		case "rc":
+			profile = "$home/lib/profile"
```
Comment (mislav) [inline commands/alias.go]: Can we suggest `~` as a shorthand for `$home`, or is it not supported in rc?

File: commands/alias.go [hunk 2]
```diff
@@ -105,6 +109,8 @@ func alias(command *Command, args *Args) {
 			eval = `function git --wraps hub --description 'Alias for hub, which wraps git to provide extra functionality with GitHub.'
 	hub $argv
 end`
+		case "rc":
+			eval = "eval `{hub alias -s}"
```
Comment (mislav) [inline commands/alias.go]: Does this syntax correctly support if the output of `hub alias -s` spans over multiple lines? In posix shells, we needed to use double quotes for that.

File: commands/alias.go (hunk 1 above)
Comment (pocket7878) [inline commands/alias.go]: ~ expansion is not supported in rc.

**Comment by mislav** [comment]
Thank you! This is great.

**Pull request merged**